### ISSUE TITLE
FlxTypedButton behave strange after scaling.

### ISSUE THREAD

**Issue by buckle2000** [HaxeFlixel/flixel#1812]
- **Flixel version:** 4.0.1
- **Affected targets: All**

---

**Code snippet reproducing the issue**:

``` haxe
package;

import flixel.FlxState;
import flixel.ui.FlxButton;

class PlayState extends FlxState
{
    override public function create():Void
    {
        var a = new FlxButton(0, 0, "asdfghjkl");
        a.scale.x*=2;
        a.scale.y*=2;
        add(a);
        // insert your code here
    }
}
```

---

**Observed behavior:** ![a picture](https://cloud.githubusercontent.com/assets/10850402/14416541/d99be9f4-ffdd-11e5-9951-ee3995f5d449.PNG)
The bounding box did not change, and text wasn't scaled.
##

**Comment by buckle2000** [comment]
`FlxSpriteButton` is the same.

**Comment by Gama11** [comment]
`scale` of a sprite doesn't automatically affect the hitbox. You can call `updateHitbox()` manually.

Automatically scaling the text would be a bit tricky regarding positioning..

Changing either of these behaviors could be a breaking change.

**Comment by buckle2000** [comment]
I think the latter one is not a breaking change. It is what FlxTypedButton supposed to be.

**Comment by Gama11** [comment]
Merged into #2047.
